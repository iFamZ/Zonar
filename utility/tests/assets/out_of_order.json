{
    "type": "out of order data - all data is out of order",
    "data": [
        {
            "ts": 1650295661,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650295660,
            "k": "longitude",
            "v": "-122.335167"
        },
        {
            "ts": 1650295655,
            "k": "speed",
            "v": "60.15"
        },
        {
            "ts": 1650295666,
            "k": "latitude",
            "v": "47.608013"
        },
        {
            "ts": 1650295663,
            "k": "longitude",
            "v": "-122.335160"
        },
        {
            "ts": 1650295680,
            "k": "latitude",
            "v": "47.608049"
        },
        {
            "ts": 1650295695,
            "k": "longitude",
            "v": "-120.335160"
        },
        {
            "ts": 1650295692,
            "k": "latitude",
            "v": "50.608049"
        }
    ]
}