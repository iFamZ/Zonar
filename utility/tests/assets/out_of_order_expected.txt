Out of order detected
Out of order detected
Out of order detected
Out of order detected
